.github/workflows/test.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Secrets check
         run: |
-          ./minnie-kenny.sh --force
+          bash -x ./minnie-kenny.sh --force
```

Comment:
Was this just experimentation?

Reply:
Note that [this change](https://github.com/broadinstitute/rawls/pull/2083/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R42) may help… it's hard to know for sure yet.

Reply:
Ugh, that commit failed on merge… back to the drawing board.